### DISCUSSION TITLE
PUG Raid sign up channel in discord

### DISCUSSION THREAD

**Discussion by jcessor** [chromiecraft/chromiecraft#7368] [Suggestions]
Like many other adult wow players i find little time to be able to commit to a consistent weekly raiding schedule as my work and family time yield varied hours to commit to raiding.

I would like to propose a dedicated ChromieCraft discord channel with the ability for people to organize pugs and allow for sign ups, in the classics retail era there were some good channels created that allowed the non-guilded community to organize and sign up for raids. 

while the existing LFG channel is great for spur of the moment group and raid finding it does not address the planning out or time constraints that raids often require. By having a seperate channel where users can generate sign ups  via Raid-Helper and creat raids beyond the boundaries of guilds all in one place. Of course guild sponsored pugs would be encouraged!

here is a link to the wow classic discord pug server i am referring to: https://discord.gg/9VqMqJmMEj (now mostly archived) 

not only do i think this would greatly improve QOL for many chromieCraft players but allow further engagement with the community and also allow for further throughput testing and bug finding of the raids.

thank you for taking the time to read my suggestion!

**Comment by sogladev** [comment]
Most pug signups happen in individual guild Discords, which have their own established rules and systems. I don't see how moving this process to the main CC discord would be practical, as someone would need to take responsibility for running and maintaining it

I do recognize that these pugs, being hosted within separate guild Discords, lack visibility. The existing 'looking-for-group' channel is available, but I see it's not actively used for recurring groups.

A new channel 'looking-for-more' can be created, that is thread-based, where recurring pugs can be listed in separate threads. Each post would provide a short advertisement for the pug along with a link to the organiser’s Discord. This would function similarly to the 'guild-recruitment' channel, but for pugs:

a. Spontaneous pugs: Organisers continue posting in the 'looking-for-group' channel and in-game chat
b. Recurring pugs: Organisers can create a thread in the 'looking-for-more' channel, with a summary and a link to their Discord

This would improve visibility for non-guilded players and allow raid organisers to reach more players who are looking for groups that fit their schedules, without creating much extra maintenance for CC staff.
